lib/redis/commands/sorted_sets.rb

```diff
@@ -167,6 +167,42 @@ def zpopmin(key, count = nil)
         end
       end
 
+      # Removes and returns up to count members with scores in the sorted set stored at key.
+      #
+      # @example Popping a member
+      #   redis.zmpop('zset')
+      #   #=> ['zset', ['a', 1.0]]
+      # @example With count option
+      #   redis.zmpop('zset', count: 2)
+      #   #=> ['zset', [['a', 1.0], ['b', 2.0]]
+      #
+      # @params key [String, Array<String>] one or more keys with sorted sets
+      # @params modifier [String]
+      #  - when `"MIN"` - the elements popped are those with lowest scores
+      #  - when `"MAX"` - the elements popped are those with the highest scores
+      # @params count [Integer] a number of members to pop
+      #
+      # @return [Array<String, Array<String, Float>>] list of popped elements and scores
+      def zmpop(*keys, modifier: "MIN", count: 1)
+        raise ArgumentError, "Pick either MIN or MAX" unless modifier == "MIN" || modifier == "MAX"
+
+        args = [:zmpop, keys.size, *keys, modifier]
+
+        if count
+          args << "COUNT"
+          args << Integer(count)
+        end
```

Comment:
```suggestion
        args << "COUNT" << Integer(count) if count
```

But also, the argument is optional on the redis server side (behave like `1` if not provided), but you default it to 1 here.

I'd day it would be more correct to default it to `nil`.